CHANGES.rst

```diff
@@ -32,6 +33,8 @@ Fixes
 Documentation
 -------------
 
+- Contribute examples to etsdemo (#380)
```

Comment:
I wasn't exactly sure the best category to place all of these.  Primarily this one, and 377.  If a reviewers opinion differs, I am happy to move them around